jwst/coron/median_replace_img.py

```diff
@@ -89,9 +90,16 @@ def median_replace_img(img_model, box_size, bad_bitvalue):
         img_int = img_model.data[nimage]
         img_dq = img_model.dq[nimage]
         bad_locations = np.where(np.bitwise_and(img_dq, bad_bitvalue))
-
+        # if it's MIRI, only use the bad locations that are in the science aperture
```

Comment:
The comment suggests that this new processing is intended only for MIRI data, but there's no logic that determines whether it's MIRI data. If it's equally OK for NIRCam (or any coronagraphic data on which this might get used at some point in the future), then perhaps the mention of MIRI should be removed. Right now the new code is in fact being applied to all inputs.

Reply:
There used to be a check for MIRI in the old filtering function, but I forgot to put it in the new one when I rewrote it to not use SIAF. I will check with the NIRCam people to see if they are OK with this behavior; otherwise I'll put it back. Thanks for checking.

Reply:
> I also wonder if anything needs to be added to the docs for the align_refs step, to mention this new behavior. That would go in /docs/jwst/align_refs/description.rst.

I don't think so, since the NON_SCIENCE pixels actually get masked out in the alignment algorithm by the PSF_MASK reference file.